src/RichTextEditor.js

```diff
@@ -154,14 +155,20 @@ export default class RichTextEditor extends Component {
         case messages.CONTENT_FOCUSED:
           this.contentFocusHandler && this.contentFocusHandler();
           break;
-        case messages.SELECTION_CHANGE:
+        case messages.SELECTION_CHANGE: {
           const items = message.data.items;
-          this.state.listeners.map((listener) => listener(items));
-          break
-        case messages.CONTENT_CHANGE:
-          const content = message.data.content;
-          this.state.onChange.map((listener) => listener(content));
-          break
+          this.state.listeners.map((listener) => {
+            listener(items);
```

Comment:
We might want to consider changing the default name `listeners` to something more descriptive